mappings/net/minecraft/registry/HolderProvider.mapping

```diff
@@ -31,3 +33,9 @@ CLASS net/minecraft/unmapped/C_pzdchrcy net/minecraft/registry/HolderProvider
 			COMMENT {@return the holder provider of the given registry, may be empty if the registry does not exist in this provider}
 			ARG 1 registryKey
 				COMMENT the key of the registry
+		METHOD m_yuymjcux get (Lnet/minecraft/unmapped/C_xhhleach;Lnet/minecraft/unmapped/C_xhhleach;)Ljava/util/Optional;
```

Comment:
```suggestion
		METHOD m_yuymjcux find (Lnet/minecraft/unmapped/C_xhhleach;Lnet/minecraft/unmapped/C_xhhleach;)Ljava/util/Optional;
```
I think `find` is more conventional when the  thing might not be found, especially when returning an `Optional`.